content/2-templating/6-conditional/marko/TrafficLight.marko

```diff
@@ -0,0 +1,17 @@
+static const TRAFFIC_LIGHTS = ["red", "orange", "green"];
+<let/lightIndex = 0/>
+<let/light = TRAFFIC_LIGHTS[lightIndex]/>
+
+<button onClick () {
+    lightIndex++;
+    if (lightIndex >= TRAFFIC_LIGHTS.length) {
+      lightIndex = 0;
+    }
```

Comment:
If we're going to do a longer version, we should do the exact same thing as the others
```suggestion
    if (lightIndex + 1 > TRAFFIC_LIGHTS.length - 1) {
      lightIndex = 0;
    } else {
      lightIndex++;
    }
```

The current version [doesn't work](https://markojs.com/playground/#NobwRAdghgtgpmAXGAlhAJnAHgOhlAJwGsB7MAGjAAcoAXACyTAHoBjEmKkiOCWgZ2ZpMufMTKV2fXrSb9adFKwAEU+coAqAJQCCAMT0BJAMIB9ADKGA4gAkNAZWUBeZcAA6YAnHQfyyjyQEUBAA5nC+-mAhXrweALoA3G4QADwANnC0zGkoIfS0hhjYzsoADMwAfMkpalk5ebQl2vpGZpa2DsD1+YUicZXJ1QBGAK60tNzK3MY5rETKABQAlMogycobyt0FRVgA1HtJEJvKKABmi9u9xRUuzQYmFtZ29jgZoQwra8cnW7k9uxKpSOJwAvutlKCKgA5bCNbYpZijcbcKqpKgVcz-RoofiIZQAEhA21BiIx1XJPwAmiQRsoYCN5BCUucnNtnE4XB4vD4wBV7BoAPIABUR5zRGxZZzZ2I5XLAgWCYQ8-PMgoA6soACIa6Fis4S5RSmUNOWRaJwWJ8qyC-VoskVMCguJAA) because the `lightIndex` doesn't get immediately updated.

Reply:
Otherwise what Luke suggested would work as well.  We just don't want to update a value and then read from it.
```suggestion
lightIndex = (lightIndex + 1) % TRAFFIC_LIGHTS.length;
```